React/Modules/RCTPointAnnotation.m

```diff
@@ -11,4 +11,6 @@
 
 @implementation RCTPointAnnotation
 
+@synthesize coordinate;
```

Comment:
I don't think it is. Someone online in my googlings had said it was necessary to auto generate the getters and setters, but I will remove it.